src/main/java/de/siegmar/logbackgelf/MessageIdSupplier.java

```diff
@@ -30,10 +30,11 @@
 
     // static random to keep Spotbugs happy
     private static final Random RANDOM = new Random();
-    private int machinePart = RANDOM.nextInt();
 
-    public int getMachinePart() {
-        return machinePart;
+    private int machinePart;
+
+    public MessageIdSupplier() {
+        machinePart = RANDOM.nextInt();
```

Comment:
Are these changes now valuable?

Reply:
They're not. Sorry for submitting them, I first reverted manually changing the code. Now I reverted this file to the upstream version using git.